types/font-carrier/index.d.ts

```diff
@@ -0,0 +1,142 @@
+// Type definitions for font-carrier 0.3
+// Project: https://github.com/purplebamboo/font-carrier#readme
+// Definitions by: zfitness <https://github.com/zFitness>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export type FontType = 'ttf' | 'eot' | 'woff' | 'woff2' | 'svg';
+
+export interface FileOptions {
+    /** 支持文件地址，也支持文件 buffer */
+    path?: string | Buffer;
+    /** 导出的字体类型 */
+    types?: FontType[];
+}
+
+export class Base<T> {
```

Comment:
correct me, if I'm wrong, but the JS module does not export any of such symbols:
https://github.com/purplebamboo/font-carrier/blob/master/lib/index.js
one should avoid creating symbols that does not exists in original. Common interface can be achiveve via, say, interface. Or just keep this private within DT module using `export {};`.

Reply:
hello, i add a base.d.ts

Reply:
please